src/sage/modules/vector_modn_dense.pyx

```diff
@@ -168,19 +168,32 @@ cdef class Vector_modn_dense(free_module_element.FreeModuleElement):
             self._init(parent.degree(), parent, parent.base_ring().order())
 
     def __init__(self, parent, x, coerce=True, copy=True):
+        """
+        Create an element.
+
+        Note that ``coerce=False`` is dangerous::
```

Comment:
```suggestion

        TESTS:

        Note that ``coerce=False`` is dangerous::
```

Reply:
Not sure if this should be test-only. The fact that "coerce=False is dangerous" feel like something that the user should know about.

Reply:
This won’t be in the public documentation, but you can make it `EXAMPLES:` if you prefer.

Reply:
Ah yes, the method is a magic method. So users reading e.g. https://doc.sagemath.org/html/en/reference/modules/sage/modules/vector_modn_dense.html won't even know `coerce=` exists at all…?

Sure, sounds reasonable.

Reply:
Yes, that's right. Users aren't supposed to really be looking at that anyways as there are other entry points. That is more for developers.